docs/blog/newsletter-february-2020.rst

```diff
@@ -0,0 +1,96 @@
+.. post:: February 03, 2020
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – February 2020
+#########################################
+
+Happy 2020, documentarians! It's Beth and the newsletter crew here, with the latest and greatest from the Write the Docs community.
+
+Two bits and pieces for you on the news front today. In December we annouced dates for the Prague conference, which will be 13-15 September - and in a brand shiny new venue, the Hotel Olšanka! `Read all about it here </conf/prague/2020/news/init/>`__. As for across the pond, the speakers for Portland speakers will be announced soon, so keep your eyes peeled for those. The conference is 3-5 May, and `tickets are on sale right now </conf/portland/2020/tickets/>`__.
```

Comment:
Maybe I'm being unduly fussy here, but the first sentence here reads a bit too colloquially (I first read "two bits" and then had to reorient myself to the rest of the sentence, and still got tripped up on "news front" ..)
Maybe trim way down, to something like "Two news items today"?
And rm "As for" in the second sentence, so it starts with "Across the pond ..."

Reply:
Or just "Some bits and pieces"

Reply:
I was clearly feeling a bit too flowery yesterday ;) thanks, fixed!